include/veriblock/pop/blockchain/pop/pop_state_machine.hpp

```diff
@@ -246,6 +246,9 @@ struct PopStateMachine {
   //! unapplies all commands in chain
   //! @overload
   void unapply(Chain<index_t>& chain) { unapply(*chain.tip(), *chain.first()); }
+  void unapply(ChainSlice<index_t>& chain) {
```

Comment:
please add doxygen `//! @overload`

Reply:
done. also added a couple of asserts to unapply() overloads